src/components/OverlappingCard/OverlappingCard.driver.ts

```diff
@@ -1,7 +1,41 @@
-import { CardDriver, cardDriverFactory } from '../Card/Card.driver';
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+  StylableUnidriverUtil,
+  UniDriver,
+} from 'wix-ui-test-utils/unidriver';
+import style from './OverlappingCard.st.css';
 
-export interface OverlappingCardDriver extends CardDriver {}
+export interface OverlappingCardDriver extends BaseUniDriver {
+  getMediaContent(): Promise<string>;
+  getInfoContent(): Promise<string>;
+  getRatio(): Promise<string>;
+  hasFlippedRatioState(): Promise<boolean>;
+  hasInvertImagePositionState(): Promise<boolean>;
+  isMediaContentExist(): Promise<boolean>;
+  isInfoContentExist(): Promise<boolean>;
+}
 
-export const overlappingCardDriverFactory: (
-  UniDriver,
-) => OverlappingCardDriver = cardDriverFactory;
+export const overlappingCardDriverFactory = (
+  base: UniDriver,
+): OverlappingCardDriver => {
+  const stylableUtil = new StylableUnidriverUtil(style);
+
+  const getMediaContainerElement = () => base.$(`.${style.mediaContainer}`);
+  const getInfoContainerElement = () => base.$(`.${style.infoContainer}`);
+
+  return {
+    ...baseUniDriverFactory(base),
+    getMediaContent: async () =>
```

Comment:
just `getNative`, not innerHTML

Reply:
@shlomitc i have to do innerHTML otherwise i get the whole container and not just the container content

Reply:
so do a css query selector to return the first child